src/ol/interaction/Draw.js

```diff
@@ -714,37 +714,39 @@ Draw.prototype.removeLastPoint = function() {
  */
 Draw.prototype.finishDrawing = function() {
   const sketchFeature = this.abortDrawing_();
-  let coordinates = this.sketchCoords_;
-  const geometry = /** @type {ol.geom.SimpleGeometry} */ (sketchFeature.getGeometry());
-  if (this.mode_ === Draw.Mode_.LINE_STRING) {
-    // remove the redundant last point
-    coordinates.pop();
-    this.geometryFunction_(coordinates, geometry);
-  } else if (this.mode_ === Draw.Mode_.POLYGON) {
-    // remove the redundant last point in ring
-    coordinates[0].pop();
-    this.geometryFunction_(coordinates, geometry);
-    coordinates = geometry.getCoordinates();
-  }
+  if (sketchFeature) {
```

Comment:
Might be more readable & less prone to future error to shorten this block.  E.g.
```js
if (!sketchFeature) {
  return;
}
// ...
```

Reply:
yes, changed